plenum/cli/cli.py

```diff
@@ -134,12 +134,14 @@ def __init__(self, looper, basedirpath, nodeReg=None, cliNodeReg=None,
                 and len(cliNodeReg)):
             self.nodeRegLoadedFromFile = True
             dataDir = self.basedirpath
-            ledger = Ledger(CompactMerkleTree(hashStore=FileHashStore(
-                dataDir=dataDir)),
+            fileHashStore = FileHashStore(dataDir=dataDir)
+            ledger = Ledger(CompactMerkleTree(hashStore=fileHashStore),
                 dataDir=dataDir,
                 fileName=self.config.poolTransactionsFile)
             nodeReg, cliNodeReg, _ = TxnStackManager.parseLedgerForHaAndKeys(
                 ledger)
+            ledger.stop()
```

Comment:
If ledger contains hashStore, shouldn't ledger.stop() should take care of stopping/closing everything  rather than we need to take care of it?